tests/views/test_views.py

```diff
@@ -0,0 +1,75 @@
+import json
+import celery
+import superset
+from superset_patchup.views import SupersetKetchupApiView
+from .. import helpers
+
+#superset.app.config['SQLALCHEMY_ECHO'] = True
+logger = celery.utils.log.get_task_logger(__name__)
+
+
+class TestApiViews:
+
+    """Tests of -ketchup API extensions"""
+
+    def setup_method(self, method):
+        helpers.cleanup_data()
```

Comment:
Could change to fixture = helpers.init_fixture() and then fixture.create_dashboard(), etc.  A little cleaner than tags, since it's not clear to me that tags will always be around.